tests/test_runnable_rails.py

```diff
@@ -51,6 +50,24 @@ def test_string_in_string_out():
     assert result == "Paris."
 
 
+def test_string_in_string_out_with_verbose_flag():
```

Comment:
We could also test a bit deeper here and look for logs made or events stored in history when `verbose` is `True`. Perhaps we should also assert that `verbose` is `False` by default?

Reply:
I think this is enough. No need to test deeper.